cfme/networks/views.py

```diff
@@ -550,6 +550,26 @@ def is_displayed(self):
                 self.title.text == '{name} (Summary)'.format(name=self.context['object'].name))
 
 
+class CloudTenantEntities(BaseEntitiesView):
```

Comment:
I think that these views are duplicating many of the views already defined in `cfme/cloud/tenant.py`, and the only unique view here is the one for a network provider's tenant collection.  Please re-use views where possible.